aide_design/physchem.py

```diff
@@ -43,16 +47,16 @@ def diam_circle(AreaCircle):
     return np.sqrt(4 * AreaCircle / np.pi)
 
 ######################### Hydraulics #########################
-RATIO_VC_ORIFICE = 0.62
+RATIO_VC_ORIFICE = con.RATIO_VC_ORIFICE
```

Comment:
replace instances of RATIO_VC_ORIFIC with con.RATIO_VC_ORIFICE at all locations in this file and take out this line.